BrowserKit/Sources/WebEngine/Engine.swift

```diff
@@ -1,10 +0,0 @@
-// This Source Code Form is subject to the terms of the Mozilla Public
-// License, v. 2.0. If a copy of the MPL was not distributed with this
-// file, You can obtain one at http://mozilla.org/MPL/2.0/
-
-import Foundation
```

Comment:
I added those files in `BrowserKit/Sources/WebEngine` in a recent PR, I guess with the move it's getting removed but they should be put back.

Reply:
I will add those back...